libs/shared/ui/src/lib/components/layouts/breadcrumb/breadcrumb.tsx

```diff
@@ -106,32 +108,46 @@ export function Breadcrumb(props: BreadcrumbProps) {
 
   const applicationMenu = [
     {
-      title: 'Applications',
+      title: 'Services',
       search: true,
       items: applications
-        ? (applications?.map((application: ApplicationEntity) => ({
-            name: application.name,
-            link: {
-              url: `${APPLICATION_URL(
-                organizationId,
-                projectId,
-                environmentId,
-                application.id
-              )}${APPLICATION_GENERAL_URL}`,
-            },
-            contentLeft: (
-              <div className="flex items-center">
-                <StatusChip status={application.status?.state} />
-                <div className="ml-3 mt-[1px]">
-                  <Icon
-                    name={!(application as ApplicationEntity).build_mode ? IconEnum.APPLICATION : IconEnum.DATABASE}
-                    width="16"
-                  />
+        ? [
+            ...(applications?.map((application: ApplicationEntity) => ({
```

Comment:
Can we have only one map for this? it's the same logic just one Icon and the link who changes?